vcluster/configure/vcluster-yaml/sync/from-host/secrets.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Secrets
+sidebar_label: secrets
+sidebar_position: 11
+description: Configuration for ...
+---
+
+import EnableSwitch from '../../../../_partials/config/sync/fromHost/secrets.mdx'
+import ProAdmonition from '../../../../_partials/admonitions/pro-admonition.mdx'
+import FromHostSecretExample from '../../../../_fragments/sync-from-host-secret-example.mdx'
+
+By default, this is turned off.
+
+:::info No need to configure RBAC
+vCluster automatically adds the required cluster RBAC permissions for retrieving the Secrets and syncing the resources from the host to the virtual cluster.
+:::
+
+Enabling this allows you to sync Secrets from the specified namespaces in the host to the specified namespaces in the vCluster.
+```yaml title="vcluster.yaml"
+sync:
+  fromHost:
+    secrets:
+      enabled: true
+      selector:
+        mappings:
+          # syncs all Secrets from "foo" namespace
+          # to the "bar" namespace in vCluster. Secrets names are unchanged.
+          "foo/*": "bar/*"
+```
+
+- It is also possible to modify the name of the synced resource in the virtual cluster.
+- There is no option to sync from all namespaces in the host.
+- Sync is one-directional, from host to virtual. If you modify an object in the host, vCluster syncs the change to virtual object.
+- When you delete a virtual object, vCluster re-creates it if the host object still exist.
+- When you delete a host object, vCluster deletes the corresponding virtual object.
+
+It is not possible to sync Secrets that were already synced from virtual to host, they are skipped by vCluster.
+
+Namespaces in the virtual cluster are created automatically during the sync (if they do not exist already).
+
+You can use synced Secrets in your workloads as an environment variables source or a volume.
+
+## Prerequisites
```

Comment:
Could we add `BasePrerequisites` partial like in the other guides? Eg here: https://github.com/loft-sh/vcluster-docs/blob/aba0e93a803729fc32947088c21173a7f71a7554/vcluster/configure/vcluster-yaml/experimental/native-sleep-mode.mdx?plain=1#L93